pkg/oci/load_balancer_security_lists_test.go

```diff
@@ -195,6 +195,37 @@ func TestGetNodeIngressRules(t *testing.T) {
 				makeIngressSecurityRule("1", 80),
 				makeIngressSecurityRule("2", 80),
 			},
+		}, {
```

Comment:
Unit test for health check port mutation. I chose to put more realistic strings in. IMHO it makes these tests easier to read/understand, especially if we added further input rules in. I can make it like the other though.